.travis.yml

```diff
@@ -8,13 +8,13 @@ jdk:
  - oraclejdk8
 
 r_github_packages:
-  - rstudio/sparklyr@2c59696
+  - rstudio/sparklyr@52461f1
```

Comment:
update to latest SparklyR with support for Spark 2.2.1

Reply:
Sounds good. We can update this to `sparklyr` without the hash once its in CRAN.

Reply:
this one is based on the officially release version, so maybe we can do just `sparklyr` now